src/init/features/firestore/indexes.ts

```diff
@@ -2,12 +2,12 @@ import * as clc from "colorette";
 import * as fs from "fs";
 
 import { FirebaseError } from "../../../error";
-import * as iv2 from "../../../firestore/indexes";
-import * as fsutils from "../../../fsutils";
+import iv2 = require("../../../firestore/api");
```

Comment:
Prefer `import * as blah from "../blah"` over `import blah = require("../blah")` - was there something that made these switches necessary?

If not, please switch back.

Reply:
yep it looks like this came from a bad merge. reverted to how you suggested